diff -up ./SkyX/Source/MeshManager.cpp.ogre19 ./SkyX/Source/MeshManager.cpp
--- ./SkyX/Source/MeshManager.cpp.ogre19	2014-06-29 13:19:03.062044456 -0400
+++ ./SkyX/Source/MeshManager.cpp	2014-06-29 13:19:03.070044624 -0400
@@ -30,7 +30,7 @@ namespace SkyX
 	MeshManager::MeshManager(SkyX *s)
 		: mSkyX(s)
 		, mCreated(false)
-		, mMesh(0)
+		, mMesh()
         , mSubMesh(0)
         , mEntity(0)
         , mVertexBuffer(0)
@@ -386,4 +386,4 @@ namespace SkyX
 
 		return cameraFarClipDistance*mRadiusMultiplier;
 	}
-}
\ No newline at end of file
+}
diff -up ./SkyX/Source/VClouds/VClouds.cpp.ogre19 ./SkyX/Source/VClouds/VClouds.cpp
--- ./SkyX/Source/VClouds/VClouds.cpp.ogre19	2014-06-29 13:19:31.534641528 -0400
+++ ./SkyX/Source/VClouds/VClouds.cpp	2014-06-29 13:19:44.116905339 -0400
@@ -53,8 +53,8 @@ namespace SkyX { namespace VClouds
 		, mGeometryManager(new GeometryManager(this))
 		, mLightningManager(new LightningManager(this))
 		, mCamerasData(std::vector<CameraData>())
-		, mVolCloudsMaterial(Ogre::MeshPtr())
-		, mVolCloudsLightningMaterial(Ogre::MeshPtr())
+		, mVolCloudsMaterial(Ogre::MaterialPtr())
+		, mVolCloudsLightningMaterial(Ogre::MaterialPtr())
 	{
 	}
 
@@ -359,4 +359,4 @@ void VClouds::setWheater(const float& Hu
 
 		mDataManager->setWheater(mWheater.x, mWheater.y, mDelayedResponse);
 	}
-}}
\ No newline at end of file
+}}
